src/containers/Tenant/Acl/Acl.scss

```diff
@@ -2,21 +2,16 @@
 
 .ydb-acl {
     width: 100%;
-    &__owner-container {
-        padding-bottom: 25px;
-    }
     &__result {
-        padding-bottom: 16px;
-    }
-
-    &__owner {
-        font-weight: 600;
+        padding-bottom: var(--g-spacing-4);
+        padding-left: var(--g-spacing-2);
     }
     &__definition-content {
         display: flex;
         flex-direction: column;
     }
-    &__group-label {
-        @include subheader-2-typography();
+    &__list-title {
+        margin: var(--g-spacing-3) 0 var(--g-spacing-5);
+        @include subheader-3-typography();
```

Comment:
It differs from titles we have in `Overview` tab. There we have `body-2-typography` and `font-size: 600`. It's defined in two places:

https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/styles/mixins.scss#L95
https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/components/InfoViewer/InfoViewer.scss#L2

I think, all these titles should be similar

![Screenshot 2024-07-22 at 12 40 19](https://github.com/user-attachments/assets/c154fd3e-86ab-46e9-88c0-8ead5449155e)

![Screenshot 2024-07-22 at 12 40 14](https://github.com/user-attachments/assets/53b02e38-c6fb-4f2a-a5f1-1d6fb1964f15)